modules/nation_notes/nn_acl_upgrade.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+* Add GACL sections for Nation Notes.
+
+* @package   LibreEHR
+* @author    Art Eaton <art@suncoastconnection.com>
+* @link      http://LibreHealth.io
+*/
+
+$ignoreAuth = true; 
+
+require_once('../../interface/globals.php');
```

Comment:
Since this is used all over the code base should ve come up with a variable like $srcdir?